mimecast/help.md

```diff
@@ -366,6 +393,7 @@ Most common cloud [URLs](https://www.mimecast.com/tech-connect/documentation/api
 
 # Version History
 
+* 3.2.0 - New action Delete Group Member
```

Comment:
You can always consider merging this PR into the other branch you have going for Mimecast once it is approved, so that you can release both new actions together in version 3.1.0 instead - if you don't do this, make sure that this plugin spec version key matches 3.2.0